classwork/05-classwork.qmd

```diff
@@ -115,6 +115,7 @@ Create a `recipe()` for the on-goal data to :
 
 -   create dummy variables
 -   remove zero-variance variables
+-   any additional steps needed for your chosen model from Day 1
```

Comment:
I dont actually think we should include this. I _dont_ think people will be using a different model than us. I expect everyone to be using the same model at this point